README.md

```diff
@@ -1,407 +1,436 @@
 
 ```lua
 local actions = require('telescope.actions')
 
-require('telescope').setup {
+-- If you want your function to run after another action you should define it as follows
+local test_action = actions._transform_action(function(prompt_bufnr)
+  print("This function ran after another action. Prompt_bufnr: " .. prompt_bufnr)
+  -- Enter your function logic here. You can take inspiration from lua/telescope/actions.lua
+end)
+
+require('telescope').setup{
   defaults = {
     mappings = {
       i = {
-        -- Disable the default <c-x> mapping
+        -- To disable a keymap, put [map] = false
+        -- So, to not map "<C-n>", just put
         ["<c-x>"] = false,
-
-        -- Create a new <c-s> mapping
-        ["<c-s>"] = actions.goto_file_selection_split,
+        -- Otherwise, just set the mapping to the function that you want it to be.
+        ["<C-i>"] = actions.goto_file_selection_split,
+        -- Add up multiple actions
+        ["<CR>"] = actions.goto_file_selection_edit + actions.center,
+        -- You can perform as many actions in a row as you like
+        ["<CR>"] = actions.goto_file_selection_edit + actions.center + test_action,
+      },
+      n = {
+        ["<esc>"] = actions.close
       },
     },
   }
 }
-```
 
-To override only SOME of the default mappings, you can use the `attach_mappings` key in the `setup` table. For example:
-
-```lua
-function my_custom_picker(results)
-  pickers.new(opts, {
-    prompt_title = 'Custom Picker',
-    finder = finders.new_table(results),
-    sorter = sorters.fuzzy_with_index_bias(),
-    attach_mappings = function(_, map)
-      -- Map "<CR>" in insert mode to the funciton, actions.set_command_line
-      map('i', '<CR>', actions.set_command_line)
-
-      return true
-    end,
-  }):find()
+-- You can also define your own functions, which then can be mapped to a key
+local function test_action(prompt_bufnr)
+  print("Action was attached with prompt_bufnr: ", prompt_bufnr)
+-- Enter your function logic here. You can take inspiration from lua/telescope/actions.lua
 end
+["<C-i>"] = test_action,
+
+-- If you want your function to run after another action you should define it as follows
+local test_action = actions._transform_action(function(prompt_bufnr)
+  print("This function ran after another action. Prompt_bufnr: " .. prompt_bufnr)
+-- Enter your function logic here. You can take inspiration from lua/telescope/actions.lua
+end)
+["<C-i>"] = actions.goto_file_selection_split + test_action
+```
+
+Configuring the mappings of a [built-in picker](#built-in-pickers)
+  is done by setting its `attach_mappings` key to a function:
+
+```lua 
+require('telescope.builtin').fd({
+  attach_mappings = function(prompt_bufnr, map)
+    map('i', '<esc>', actions.close)
+  end
+})
+```
+
+
+## Built-in Pickers
+
+Built-in function ready to be bound :D.
+
+| Functions                           | Description                                                      | Status |
```

Comment:
Is Status really necessary? It kinda indicates that some `builtin`s are not done and can't be used.

Reply:
Done